src/server/game.js

```diff
@@ -62,11 +62,11 @@ export default class Game extends EventEmitter {
     let id = this._availableSeats.shift();
     this.playersInGame++;
 
-    if(id !== undefined) {
+    if (id !== undefined) {
       let player = new Player({
-        socket: socket,
-        id: id,
-        nickname: 'Player ' + (id.substr(-1)),
+        socket,
+        id,
```

Comment:
'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).